abe/fame.go

```diff
@@ -120,6 +121,39 @@ func (a *FAME) Encrypt(msg string, msp *MSP, pk *FAMEPubKey) (*FAMECipher, error
 
 	}
 
+	// msg is encrypted using CBC, with a random key that is encapsulated
+	// with GPSW
```

Comment:
In FAME, the random key is encapsulated with GPSW?

Reply:
Copy paste mistake:)